app/services/archive_auction.rb

```diff
@@ -17,4 +18,13 @@ def perform
   def self.archive_submit?(params)
     params.key?(:archive_auction)
   end
+
+  private
+
+  def update_c2_status
+    return unless auction.c2_proposal_url.present?
```

Comment:
oof I am not a fan of this style (I know many are). I feel like this is so much clearer:

```
  if auction.c2_proposal_url.present?
    UpdateC2ProposalJob.perform_later(auction.id, 'C2CancelAttributes')
    auction.c2_status = :c2_canceled
end
```

what do you think?